src/api/dependency-discovery/Dockerfile

```diff
@@ -16,18 +16,18 @@ RUN pnpm install --prod
 
 ## Deploy ########################################################################
 # Stage for running our app
-FROM base as deploy
+FROM node:16-alpine3.15 as deploy
 
```

Comment:
Add a `WORKDIR <DIRECTORY>`, instead of having everything in the root.
The same thing applies to the rest of Dockerfile.

Reply:
So in here https://github.com/humphd/telescope/blob/3f6af21e4dd3a968e315e408c5e60719026aa5c3/src/api/feed-discovery/Dockerfile#L19-L25
The code will be copied into the root folder instead of being in a folder that holds our app.

We can change into 
```dockerfile
WORKDIR /app

COPY --chown=node:node . .

COPY --chown=node:node --from=dependencies /app/node_modules ./node_modules
```
or 
```dockerfile

COPY --chown=node:node . /app

COPY --chown=node:node --from=dependencies /app/node_modules /app/node_modules
```